e2e/cypress/integration/markdown/markdown_image_spec.js

```diff
@@ -11,8 +11,13 @@ import * as TIMEOUTS from '../../fixtures/timeouts';
 
 describe('Markdown', () => {
     before(() => {
-        cy.apiLogin('user-1');
-        cy.visit('/');
+        // # Login as new user
+        cy.loginAsNewUser().then(() => {
+            // # Create new team and visit its URL
+            cy.apiCreateTeam('test-team', 'Test Team').then((response) => {
+                cy.visit(`/${response.body.name}`);
+            });
+        });
```

Comment:
This spec is not broken but I realized that this is good to isolate to new user since this was originally posting large capacity images which might affect succeeding test cases to wait longer than usual.

Reply:
I see these steps are used in multiple testcases like markdown_image_spec and markdown_text_spec. Instead of having these steps in every testcase, can we create a common function in api_commands.js and use it across.

Reply:
@saturninoabril I think this is a great idea, all the large images do (or did?) impact the other tests.

Reply:
@srkgupta I'm reorganizing how we are doing API things, I'll make something like that so we can give it a whirl.

Reply:
Ok sure. Will go ahead and accept this then. Thanks for letting me know @thekiiingbob

Reply:
@srkgupta That's really good point but let's have it organize together with what @thekiiingbob is currently working on.